tests/drivers/sensor/sbs_gauge/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+set(CONF_FILE "proj.conf")
+
+if(${BOARD} STREQUAL "nucleo_f070rb")
+  set(DTC_OVERLAY_FILE
+    "${DTC_OVERLAY_FILE} ${CMAKE_CURRENT_LIST_DIR}/dts/overlays/nucleo_f070rb.overlay"
+    CACHE STRING "" FORCE
+    )
+endif()
```

Comment:
Just place the overlay in `boards/nucleo_f070rb.overlay` relative to the application directory; the build system will pick it up by default.

Reply:
moved the overlay as proposed, thx for the hint